hack/jenkins/e2e.sh

```diff
@@ -364,6 +364,33 @@ case ${JOB_NAME} in
     ADMISSION_CONTROL="NamespaceLifecycle,InitialResources,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota"
     ;;
 
+  # Runs only the ingress tests on GCE.
+  kubernetes-e2e-gce-ingress)
+    : ${E2E_CLUSTER_NAME:="jenkins-gce-e2e-ingress"}
+    : ${E2E_NETWORK:="e2e-ingress"}
+    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=\[Feature:Ingress\]"}
+    : ${KUBE_GCE_INSTANCE_PREFIX:="e2e-ingress"}
+    : ${FAIL_ON_GCP_RESOURCE_LEAK:="true"}
+    # TODO: Move this into a different project. Currently, since this test
```

Comment:
After the large features we've added to it have stabilized I'd like to add it to the regular project. Actually for now it's as much to isolate my debugging from leaks in l4 loadbalancer tests as it is the reverse. I can create a kubernetes-ingress project and bump up the quota. I figured it's not that big a deal, you prefer that?